### PR TITLE
Add HPCC mpi-only to benchpark

### PR DESCRIPTION
**Pull request by rfhaque**
Tested on cts1

### CODE REVIEW

**Comment by pearce8** [comment]
@rfhaque Please test on its-1 using the x86 config.

File: configs/cts1/auxiliary_software_files/packages.yaml
Comment (pearce8) [inline configs/cts1/auxiliary_software_files/packages.yaml:1]: Please remove fftw as a handle expected from a system.  We should use Spack to build it instead.

File: experiments/hpcc/mpi-only/ramble.yaml
Comment (pearce8) [inline experiments/hpcc/mpi-only/ramble.yaml:1]: environments:
       hpcc:
         packages:


Should not have fftw.

**Review by pearce8 — CHANGES_REQUESTED** [review]

File: configs/cts1/auxiliary_software_files/packages.yaml
Comment (rfhaque) [reply configs/cts1/auxiliary_software_files/packages.yaml:1]: @pearce8 Removed fftw

**Comment by rfhaque** [comment]
> @rfhaque Please test on its-1 using the x86 config.

Tested on cts1 with the x86 config

**Review by pearce8 — APPROVED** [review]

**Pull request merged**